### PR TITLE
'Solution'

### PR DESCRIPTION
**Pull request by b-antonenko**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+<script src="js.js"></script>
+</body>
+</html>
```
Comment (gestap) [inline index.html]: the document must end with one empty line

File: js.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+class Todo {
+    constructor() {
```
Comment (gestap) [inline js.js]: use 2 whitespaces instead of 4

File: js.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        this.list.push(newObj);
+        return newObj.id;
+    }
+
+    removeItem(id) {
+        let elIndex = this.list.findIndex(newObj => newObj.id ===id);
+        if (elIndex !== -1) {
+            this.list.splice(elIndex, 1);
+            return true;
+        } return false;
```
Comment (gestap) [inline js.js]: https://github.com/mate-academy/style-guides/blob/master/javascript.md#whitespace--after-blocks

File: js.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    }
+
+    getItem(id) {
+        let foundObj = this.list.find(newObj => newObj.id === id);
+        return foundObj || null;
+    }
+
+    next() {
+        try {
```
Comment (gestap) [inline js.js]: no need for `try...catch`

File: js.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+            }
+            this.list.sort((a, b) => {
+                if(a.priority < b.priority) return 1;
+                if(a.priority > b.priority) return -1;
+                if(a.priority === b.priority) return 0;
+            });
+            return this.list[0];
+        } catch(error) {
+            console.log('error');
+            return false;
```
Comment (gestap) [inline js.js]: it must `throw an error` instead of `return: false`

File: js.js [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+            title : title,
+            priority : priority,
+            id : this.id++
+        }
+        this.list.push(newObj);
+        return newObj.id;
+    }
+
+    removeItem(id) {
+        let elIndex = this.list.findIndex(newObj => newObj.id ===id);
```
Comment (gestap) [inline js.js]:
```suggestion
        let elIndex = this.list.findIndex(newObj => newObj.id === id);
```

**Review by ost-ptk — CHANGES_REQUESTED** [review]

File: js.js [hunk 7]
```diff
@@ -0,0 +1,46 @@
+'use strict';
+
```
Comment (ost-ptk) [inline js.js:2]: use 2 whitespaces instead of 4 in all document
https://github.com/mate-academy/style-guides/blob/master/javascript.md#whitespace--spaces

**Review by ost-ptk — CHANGES_REQUESTED** [review]

**Review by ost-ptk — APPROVED** [review]

**Pull request closed without merging**